content/podcast/Cultural-Evolution-Implications-and-whats-next.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Joseph Henrich and Cultural Evolution: Implications and what's next"
+author: lauren-wigmore
+date: 2023-09-10
+src: https://anchor.fm/s/????.m4a
+---
+
+In Episode 4 of the series Joseph Henrich and Rufus Pollock continue on the Culture Evolution conversation.
+
+In this second part of the conversation continuing on directly from episode 3, Rufus Pollock and Joe Henrich discuss the implications of cultural evolution for modern challenges. They explore the potential for intentional experimentation in creating cultural norms that promote trust, cooperation, group cohesion and a sense of community and belonging. 
+
+Rufus and Joe touch upon the idea that Western societies might be running on the fumes of values and norms that were cultivated by historical religious practices. They discuss the need to find ways to renew and revitalize these values, potentially by experimenting with intentional communities that incorporate elements of shared meaning, trust-building, ritual and cooperation. The talk moves on to discuss intentional communities and suggest that by allowing a variety of intentional communities to form and observing which ones thrive, societies could potentially find ways to address challenges and promote positive cultural evolution.
+
```

Comment:
Add "This conversation forms part of the Cultural Evolution: A New Discipline is Born Series." and link to learn page